### DISCUSSION TITLE
Possible bug: disabling automatic backups in admin does not cancel current cron

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#4431] [Q&A]
I think the current auto-backup cron keeps running even after setting it to disabled (and saving) in the admin.

Test:
* Set auto-backup cron to `*/5 * * * *` and save. Restart pocketbase just to be sure
* Allow run for 10 minutes, observe two new entries in backup log
* Toggle auto-backup to disabled & save
* Wait 10 more minutes, observe another two new entries in backup log. New entries continue every 5 minutes

<img width="740" alt="image" src="https://github.com/pocketbase/pocketbase/assets/1068356/03e9cc23-f063-4489-92a5-7a082a2cc62c">

**Comment by ganigeorgiev** [comment]
The scheduled job is refreshed on db settings save but the problem was that `app.Settings()` is updated after the db persistence and when we reload the cron job it was still holding the old value.

A "fix" was submitted in the [`develop` branch](https://github.com/pocketbase/pocketbase/commit/53ee5212bc0fe9f12fbca88b420d61abc4a8f1e1) and it will be available with the next v0.22.0 release (most likely sometime later this week). A temporary workaround if you don't want to wait could be to save twice the settings.

_Side-note: If you think that there is a bug directly open an issue with the steps to reproduce as there is nothing to discuss in this case._
